backend/organization/permissions.py

```diff
@@ -224,4 +224,12 @@ def has_permission(self, request, view):
         if requesting_member:
             return True
 
-        return False
\ No newline at end of file
+        return False
+
+class ApproveDenyProjectMemberRequest(BasePermission):
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
+            return True
+        else:
+            project = Project.objects.filter(url_slug=str(view.kwargs.get('project_slug'))).first()
+            return ProjectMember.objects.filter(project=project,user=request.user, role__role_type__in=[Role.ALL_TYPE]).exists()
```

Comment:
Shouldn't the role type only admins to approve or deny project memebership?

Reply:
That makes sense to me! I'll update to handle this change 👍

Reply:
Ok coming back here - this might be misunderstanding the code, but I interpret this as that's actually the behavior, looking at: https://github.com/climateconnect/climateconnect/blob/master/backend/climateconnect_api/models/role.py#L41